packages/extension/src/manager/contexts-states-dispatcher.ts

```diff
@@ -77,6 +80,10 @@ export class ContextsStatesDispatcher {
   }
 
   async dispatch(channel: RpcChannel<unknown>, options?: unknown): Promise<void> {
+    if (!this.hasSubscribers(channel.name)) {
+      return;
+    }
+    console.debug('dispatch data for', channel.name);
```

Comment:
keep ?

Reply:
yes, I would like to keep it for the moment